docs/releasenotes/6.0.0.rst

```diff
@@ -32,6 +32,14 @@ Removed deprecated VERSION
 ``VERSION`` (the old PIL version, always 1.1.7) has been removed. Use ``__version__``
 instead.
 
+New File Format
+===============
+
+DIB File Format
```

Comment:
Shall we put this under "API Additions" (or "API Changes?") instead of a new header?

Reply:
My thinking was to copy https://github.com/python-pillow/Pillow/blob/master/docs/releasenotes/5.1.0.rst#blp-file-format.

I have moved it to API Additions.